lib/src/components/NxCombobox/NxCombobox.scss

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../../scss-shared/nx-container-helpers';
+
+.nx-combobox {
+  display: inline-block;
+  position: relative;
+
+  &__menu {
+    position: fixed;
+    right: 100%;
+    width: 100%;
+  }
+
+  &__input {
+    margin-bottom: 0;
+  }
+
+  &__alert {
+    @include nx-container-helpers.container-vertical;
+    position: fixed;
+    right: 100%;
+    width: 100%;
+    border-radius: var(--nx-border-radius);
+    margin: var(--nx-spacing-base) 0 0 0;
+    padding: var(--nx-spacing-base) 0; 
+    background: var(--nx-swatch-white);
+    box-shadow: 0 2px 12px 0 rgb(74 77 113 / 20%), 0 1px 2px 0 rgb(91 95 141 / 20%), 0 1px 1px 0 rgb(124 128 179 / 20%);
```

Comment:
What is the complicated box shadow? I don't see anything like this elsewhere in RSC's styles

Reply:
I would like to have the same style of dropdown on alert block, so was referring to the CSS in NxDropdownMenu.scss  
` box-shadow: 0 2px 12px 0 rgba(74, 77, 113, 0.2),`
                          `0 1px 2px 0 rgba(91, 95, 141, 0.2),`
                          `0 1px 1px 0 rgba(124, 128, 179, 0.2);`

Reply:
> so was referring to the CSS in NxDropdownMenu.scss

Ah! My grep didn't find it because it's split over multiple lines.  In that case, let's move this value to a CSS variable in `_nx-variables.scss`.

Reply:
It is updated here a59e762c3a1d648b3d67c8e5a9ec4e7ff9ec06ce.